pycaching/cache.py

```diff
@@ -729,6 +810,16 @@ def load_by_guid(self):
         self.location = Point.from_string(
             content.find("p", "LatLong Meta").text)
 
+        self.lat = str(content.find(id="ctl00_ContentBody_Location").find("a").find("a")).split("=")[2].split("&")[0]
```

Comment:
None of the bellow can work, because the content is loaded from cache print-page ([example](https://www.geocaching.com/seek/cdpf.aspx?guid=40041302-7f06-498c-a236-0fed7b2dbf3d)), which doesn't have these elements. So I would like to kindly ask you to delete these lines, irrespectively of what has been written in other comments.